tests/sync/test_context_manager.py

```
@@ -72,12 +72,42 @@ async def test_long_history_truncated(self):
+        chat_history = []
+        # We need an odd number of messages before the keep window so the
+        # slice boundary falls on an assistant message.
+        total_pairs = (DEFAULT_MESSAGES_TO_KEEP // 2) + 5
+        for i in range(total_pairs):
+            chat_history.append({"role": "user", "content": f"Message {i}"})
+            chat_history.append({"role": "assistant", "content": f"Response {i}"})
+        # Add one extra user message so the -DEFAULT_MESSAGES_TO_KEEP index
+        # falls on an assistant message
+        chat_history.append({"role": "user", "content": "Extra"})
+
+        history, full_history, summary = await prepare_history_for_llm(
```

Ruff (0.15.1)

[warning] Unpacked variable `full_history` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)